app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    line = command.split()
+
+    first_file = line[1]
+    second_file = line[2]
```

Comment:
Consider using more descriptive variable names, such as `source_file` and `destination_file`, to improve code readability and adhere to the code style guidelines.